guides/v2.1/test/integration/annotations/magentoAppArea.md

```diff
@@ -0,0 +1,112 @@
+---
+group: integration-testing
+version: 2.1
+title: Application Area Annotation in the Integration Testing Framework
+github_link: test/integration/annotations/magentoAppArea.md
+---
+
+Integration testing framework enables you to configure the application area to run tests using the `@magentoAppArea` annotation.
+
+## Format
+
+> Configuring the test environment in scope of the specified application area
+
+```php?start_inline=1
+/**
+ * @magentoAppArea <areaCode>
+ */
+```
+
+## Usage
+
+The annotation is applied following the fallback scheme below:
+
+1. Test annotation
+2. Test case annotation
+3. Default application area, which is `global`
+
+### In a test case
+
+The annotation in a test case enables the specified application area for all tests in the test case.
+If the annotation is specified over a test in the test case, it will overwrite the test case annotation.
+
+> The test case annotation example:
+
+```php?start_inline=1
+/**
+ * @magentoAppArea adminhtml
+ */
+class Some_Class_ToTest extends PHPUnit_Framework_TestCase
```

Comment:
Please replace with namespaced `PHPUnit\Framework\TestCase` (Magento already switched to a version of PHPUnit with namespaced classes)

Reply:
🆗